components/Slider/README.md

```diff
@@ -215,3 +220,34 @@ id<MDCColorScheming> colorScheme = [[MDCSemanticColorScheme alloc] init];
 ```
 <!--</div>-->
 
+
+## Accessibility {a11y}
+
+<!-- Extracted from docs/accessibility.md -->
+
+## Accessibility {a11y}
+
+To help ensure your slider is accessible to as many users as possible, please be sure to review the following recommendations:
+
+### Set `-accessibilityLabel`
```

Comment:
Consider naming the section headers after the property names and omit the formatting. I think the header should take care of the formatting in this case.